src/graphics/Screen.cpp

```diff
@@ -775,10 +778,8 @@ static void drawNodeInfo(OLEDDisplay *display, OLEDDisplayUiState *state, int16_
             else
                 snprintf(distStr, sizeof(distStr), "%.1f km", d / 1000);
 
-            // FIXME, also keep the guess at the operators heading and add/substract
-            // it.  currently we don't do this and instead draw north up only.
             float bearingToOther =
-                GeoCoord::bearing(DegD(p.latitude_i), DegD(p.longitude_i), DegD(op.latitude_i), DegD(op.longitude_i));
```

Comment:
While this was technically working it was only because of the axis flips that happened when drawing the direction arrow on the compass. For clarity the points used for calculating bearing have been reversed to be consistent with the name of the value the function is returning to "bearingToOther".